types/rule.pp

```diff
@@ -1,7 +1,7 @@
 type Patterndb::Rule = Struct[
   {
     patterns        => Array[String[1]],
-    id              => String[1],
+    id              => Optional[String[1]],
```

Comment:
I guess that all the Struct I have ever created are wrong and should be fixed from `foo => Optional[bar]` to `Optional[foo] => bar` :unamused:

Reply:
I replaced all old-wrong-old-style optionals

Reply:
Awesome!